MovieViewer/Movie.swift

```diff
@@ -8,19 +8,47 @@
 
 import Foundation
 
+var favoritedMovies = [Int]()
```

Comment:
sorry, should comment here on the PR, not the commit. this might be better off as `static`.